TF_Tooling_Web/Plugins/SnappingPlugin/Snapping_Example.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
     <meta charset="UTF-8">
-    <title>Quick Start Example</title>
-    <script name="TouchFree_Tooling" type="text/javascript" src="../dist/TouchFree_Tooling.js"></script>
-    <script name="TouchFree_Tooling" type="text/javascript" src="../dist/Snapping_Plugin.js"></script>
-    <script type="text/javascript" src="./TouchFree_Tooling_Quick-Start.js"></script>
+    <title>Snapping Plugin Example</title>
+    <script name="TouchFree_Tooling" type="text/javascript" src="../../dist/TouchFree_Tooling.js"></script>
+    <script name="TouchFree_Tooling" type="text/javascript" src="./Snapping_Plugin.js"></script>
```

Comment:
`Snapping_Plugin.js` isn't found because the build name is `SnappingPlugin.js` without the underscore. 
Should we add the underscore for consistency with `TouchFree_Tooling.js` ?